icd/api/vk_cmdbuffer.cpp

```diff
@@ -7509,7 +7509,7 @@ void CmdBuffer::BindTargets(
 
         PalCmdBuffer(deviceIdx)->CmdBindTargets(params);
 
-        if (pRenderingFragmentShadingRateAttachmentInfoKHR != nullptr)
+        if (pRenderingFragmentShadingRateAttachmentInfoKHR != nullptr && pRenderingFragmentShadingRateAttachmentInfoKHR->imageView != VK_NULL_HANDLE)
```

Comment:
Each expression needs () around it.

The line exceeds the 120 character limit.